herbert_core/applications/multifit/@mfclass/private/indices_parse.m

```diff
@@ -0,0 +1,55 @@
+function ind = indices_parse (ind_in,ndatatot,str)
+% Check that a list of dataset indices has the correct format valid values
+%
+%   >> ind = indices_parse (ind_in,ndatatot,str)
+%
+% Input:
+% ------
+%   ind_in      List of dataset indices (row vector)
+%               This function checks that all indices are in the range 1
+%              to ndatatot, and that there are no repeated indices
+%               Can be empty (i.e. []); this is valid even if ndatatot=0
+%               If 'all' then will be interpreted as all datasets
```

Comment:
I don't understand this line. I think it should be
"If ind_in=='all' then 1:ndatatot will be returned unless ndatatot==0, in which case [] will be returned."